test/functional/autocmd/termxx_spec.lua

```diff
@@ -90,6 +90,18 @@ describe('autocmd TermClose', function()
     retry(nil, nil, function() eq('3', eval('g:abuf')) end)
     feed('<c-c>:qa!<cr>')
   end)
+
+  it('exposes v:event.status', function()
+    -- command('set shell="'..nvim_dir..'/shell-test" EXIT')
+    command('set shellcmdflag=EXIT shellredir= shellpipe= shellquote= shellxquote=')
```

Comment:
Windows CI is passing, guess one of the default `shell*` options was sending some input.

Reply:
Nice, good work.